pkg/registry/rbac/rolebinding/policybased/storage.go

```diff
@@ -46,6 +47,14 @@ func (s *Storage) Create(ctx api.Context, obj runtime.Object) (runtime.Object, e
 		if s.superUser != "" && user.GetName() == s.superUser {
 			return s.StandardStorage.Create(ctx, obj)
 		}
+
+		// system:masters is special because the API server uses it for privileged loopback connections
```

Comment:
@liggitt and I spoke at length about it.  He doesn't think it looks pretty, but couldn't come up with a reason to avoid this and makes the bootstrapping flow very clean in comparison to what we did in openshift.